test/User-test.js

```diff
@@ -175,7 +175,7 @@ describe('User', function() {
     const updatedFlourAmount = user.pantry[5].amount;
 
     expect(originalFlourAmount).to.equal(9);
-    expect(updatedFlourAmount).to.equal(8)
```

Comment:
Here I think instead of changing the expected value here, we need to remove the negative sign from like 169, so that the it subtracts 1 from the amount.